File: C# PowerPoint Examples/NotesSlidesApiDemo.cs
using Spire.Cloud.Powerpoint.Sdk.Client;
using Spire.Cloud.Powerpoint.Sdk.Api;

namespace CloudPowerpoint
{
    class NotesSlidesApiDemo
    {
        static string appId = "your id";
        static string appKey = "your key";
        static string baseUrl = "https://api.e-iceblue.cn";
        static Configuration configuration = new Configuration(appId, appKey, baseUrl);
        static NotesSlidesApi notesSlidesApi = new NotesSlidesApi(configuration);
        public static void DeleteNotesSlide()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string storage = null;
            string folder = "input";
            var response = notesSlidesApi.DeleteNotesSlide(name, slideIndex, password, folder, storage);
        }
        public static void GetNotesSlide()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string password = null;
            string storage = null;
            string folder = "input";
            var response = notesSlidesApi.GetNotesSlide(name, slideIndex, password, folder, storage);
        }
        public static void AddNotesSlide()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string note = "This is note info";
            string password = null;
            string storage = null;
            string folder = "input";
            var response = notesSlidesApi.AddNotesSlide(name, slideIndex, note, password, folder, storage);
        }
        public static void SetNotesSlide()
        {
            string name = "template.pptx";
            int? slideIndex = 0;
            string note = "update note";
            string password = null;
            string storage = null;
            string folder = "input";
            var response = notesSlidesApi.SetNotesSlide(name, slideIndex, note, password, folder, storage);
        }
    }
}

File: Java PowerPoint Examples/NotesSlidesApiDemo.java
import spire.cloud.powerpoint.sdk.*;
import spire.cloud.powerpoint.sdk.api.NotesSlidesApi;
import spire.cloud.powerpoint.sdk.model.NotesSlide;

public class NotesSlidesApiDemo {
    static String appId = "your id";
    static String appKey = "your key";
    static String baseUrl = "https://api.e-iceblue.cn";
    static Configuration configuration = new Configuration(appId, appKey, baseUrl);
    static NotesSlidesApi notesSlidesApi = new NotesSlidesApi(configuration);

    public static void deleteNotesSlide() throws ApiException {
        String name = "template.pptx";
        Integer slideIndex = 0;
        String password = null;
        String storage = null;
        String folder = "input";
        NotesSlide response = notesSlidesApi.deleteNotesSlide(name, slideIndex, password, folder, storage);
    }

    public static void getNotesSlide() throws ApiException {
        String name = "template.pptx";
        Integer slideIndex = 0;
        String password = null;
        String storage = null;
        String folder = "input";
        NotesSlide response = notesSlidesApi.getNotesSlide(name, slideIndex, password, folder, storage);
    }

    public static void addNotesSlide() throws ApiException {
        String name = "template.pptx";
        Integer slideIndex = 0;
        String note = "This is note info";
        String password = null;
        String storage = null;
        String folder = "input";
        NotesSlide response = notesSlidesApi.addNotesSlide(name, slideIndex, note, password, folder, storage);
    }

    public static void setNotesSlide() throws ApiException {
        String name = "template.pptx";
        Integer slideIndex = 0;
        String note = "update note";
        String password = null;
        String storage = null;
        String folder = "input";
        NotesSlide response = notesSlidesApi.setNotesSlide(name, slideIndex, note, password, folder, storage);
    }
}
